apps/web/src/pages/InstructorLedTrainingPage/InstructorLedTrainingPage.tsx

```diff
@@ -0,0 +1,545 @@
+                            size="h5"
+                            data-h2-font-weight="base(700)"
+                            data-h2-margin="base(0)"
+                            data-h2-text-align="base(center) l-tablet(initial)"
+                          >
+                            {localizedTitle}
+                          </Heading>
+                          <CourseLanguageChip
+                            courseLanguage={
+                              trainingOpportunity.courseLanguage?.value
+                            }
+                          />
+                        </div>
+                        <MetaData
+                          metaData={getMetaData(intl, trainingOpportunity)}
+                          data-h2-align-items="base(center)"
+                        />
+                        <CardSeparator />
+                        <Dialog.Root>
+                          <Dialog.Trigger>
+                            <Button mode="text" data-h2-font-weight="base(700)">
+                              {intl.formatMessage({
+                                defaultMessage: "Learn more and apply",
+                                id: "k6HT4M",
+                                description:
+                                  "Button label to open a training opportunities dialog",
+                              })}
+                            </Button>
+                          </Dialog.Trigger>
+                          <Dialog.Content>
+                            <Dialog.Header>{localizedTitle}</Dialog.Header>
+                            <Dialog.Body>
+                              <RichTextRenderer
+                                node={htmlToRichTextJSON(
+                                  getLocalizedName(
+                                    trainingOpportunity.description,
+                                    intl,
+                                  ),
+                                )}
+                              />
+                              <Dialog.Footer>
+                                <Link
+                                  href={getLocalizedName(
+                                    trainingOpportunity.applicationUrl,
+                                    intl,
+                                  )}
+                                  newTab
+                                  color="secondary"
+                                  mode="solid"
+                                >
+                                  {intl.formatMessage({
+                                    defaultMessage: "Apply now",
+                                    id: "PH68o/",
+                                    description:
+                                      "Label for apply now button in instructor led training",
+                                  })}
+                                </Link>
+                                <Dialog.Close>
+                                  <Button mode="text" color="tertiary">
```

Comment:
I believe this should be "warning" colour

Reply:
Fixed in [b0552be](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12083/commits/b0552bec13d26b2ce5a2ee8b76506e14b3fa8421)